cmake/Modules/FindClangFormat.cmake

```diff
@@ -0,0 +1,12 @@
+INCLUDE(FindPackageHandleStandardArgs)
+FIND_PROGRAM(CLANGFORMAT_EXECUTABLE clang-format HINTS ${CLANGFORMAT_ROOT})
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(clang-format DEFAULT_MSG CLANGFORMAT_EXECUTABLE)
```

Comment:
Usualy find_package_handle_standard_args is putted at the end of script.

It's common place where others will search it.